<!DOCTYPE html><html lang="en"><head><title>TAS SRS Abstracts - 2016</title><link rel="shortcut icon" href="http://www.ramapo.edu/tas/wp-content/themes/opamar/images/favicons/favicon.ico"><link href="https://fonts.googleapis.com/css?family=Roboto:400,700,400italic" rel="stylesheet" type="text/css"><style type="text/css">@media only screen and (min-width: 801px) {
  .altheader {
display:none;
}
}

@media print {
      .abstract,.full  {page-break-after: always;}
      .contents, a {display:none;}
  }
@media only screen and (max-width: 800px) {
      img {
          display: none;
      }
      ul
          {
          list-style-type: none;
          padding-left:0px;
          }
      li {
          padding:0px;
      }
      body {
          padding:0.25em;
      }
      .altheader {
      display:block;
      }
  }
.type {
  font-style:italic
}
body {
  padding:1em;  
  font-family: 'Roboto', sans-serif;
}
.abstract {
  margin-bottom:3em;
}
.contents{
  font-size:smaller;
}
li {
  margin-bottom:1em;
}
a, a.visited {
  color: #8b0000;
}
a {
  text-decoration:none
}
.header {
  text-align:center
}
.full {
  margin-top:5em;
}
</style></head><body><div class="header"><img src="http://www.ramapo.edu/tas/files/2012/07/SRS-2016-banner-768x300.png"></div><div class="altheader"><h1>2016 TAS Student Research Symposium</h1></div><div class="contents"> <h2>Table of Contents</h2><a name="contents"></a><ul><li><p><span><a href="#0">A NATIVE WILDFLOWER MEADOW AT RAMAPO COLLEGE OF NEW JERSEY: ESTABLISHMENT, MAINTENANCE AND DATA FROM THE FIRST TWO YEARS</a></span><br><span>Melissa Mayberry (Senior, Bioinformatics &amp; Computer Science)</span></p></li><li><p><span><a href="#1">AC-MAGNETIC SUSCEPTIBILITY OF AMORPHOUS MAGNETIC ALLOYS</a></span><br><span>Nathaniel Birrer (Junior, Engineering Physics), Patrick Rehain (Junior, Physics), Kamil Nowak (Junior, Engineering Physics), Christopher Bolton (Senior, Engineering Physics)</span></p></li><li><p><span><a href="#2">ANALYSIS OF SIT EXPRESSION ON A PROTEIN BASED LEVEL</a></span><br><span>Taylor  Donnelly (Junior, Biology)</span></p></li><li><p><span><a href="#3">ANDROID APPLICATION TO AID BIRD IDENTIFICATION IN THE FIELD</a></span><br><span>Thomas Harden (Senior, Biology)</span></p></li><li><p><span><a href="#4">ANTIBIOTIC RESISTANCE AMONG ISOLATES OF STAPHYLOCOCCUS AUREUS FROM RAMAPO COLLEGE STUDENTS</a></span><br><span>David Cifelli (Sophomore, Biology), Sophia Penkrat, Amber Smith</span></p></li><li><p><span><a href="#5">BACTERIAL GROWTH STUDIES OF GUT MICROBES INCLUDING LACTOBACILLUS RHAMNOSUS GG AND ESCHERICHIA COLI HS USING UV-VIS SPECTROPHOTOMETRY AND QUANTITATIVE PCR (QPCR)</a></span><br><span>Devashri Parikh (Sophomore, Biochemistry), Pritha Aggarwal (Sophomore, Biochemistry), Katerina Djambazova (Sophomore, Biochemistry, Psychology)</span></p></li><li><p><span><a href="#6">COMPARATIVE ANALYSIS OF PHASEOLUS VULGARIS AND CICER ARIENTIUM CONVENTIONALLY VERSUS VERTICAL HYDROPONICALLY GROWN</a></span><br><span>Brenda Arthur (Junior, Biology), Stephanie  Morfi (Senior, Biology)</span></p></li><li><p><span><a href="#7">COMPARING FOOD KNOWLEDGE AND DIET HABIT OF COLLEGE STUDENTS IN US AND CHINAS METROPOLITAN AREAS</a></span><br><span>Brenda Arthur (Junior, Biology), Stephanie  Morfi (Senior, Biology)</span></p></li><li><p><span><a href="#8">DEVELOPMENT OF A BIOINFORMATICS APPLICATION FOR TRANSCRIPTOME WIDE GENE FUNCTION ANALYSIS OF MRNAS WITH MAPPED CIS-REGULATORY G-QUADRUPLEX MOTIFS.</a></span><br><span>Melissa Mayberry (Senior, Bioinformatics &amp; Computer Science)</span></p></li><li><p><span><a href="#9">DEVELOPMENT OF CAPABILITY FOR PRODUCTION OF MONOCLONAL ANTIBODIES</a></span><br><span>Jennifer Howard (Junior, Biology)</span></p></li><li><p><span><a href="#10">DEVELOPMENT OF LENTIVIRAL DELIVERY OF OVEREXPRESSION AND SILENCING VECTORS TO PRIMARY CELLLS</a></span><br><span>Kate  Reilly (Senior, Biology), Sara Luby (Senior, Biology)</span></p></li><li><p><span><a href="#11">DEVELOPMENT OF SILENCING TOOLS FOR SRC PROTOC-ONGOGENE, NON RECEPTOR TRYOSINE GENE (C-SRC) AND SIGNALING THRESHOLD REGULATING TRANSMEMBRANE ADAPTOR 1 GENE (SIT1)</a></span><br><span>Nicole Nebbia (Senior, Biology), Donna Romhen (Senior, Biology), Emma Murray (, )</span></p></li><li><p><span><a href="#12">DOES CHRONIC EXPOSURE TO TRICLOSAN LEAD TO RESISTANCE TO UNRELATED ANTIBIOTICS?</a></span><br><span>Alexandria Farrell (Senior, Biology), Hailey Ebue (Junior, Biology)</span></p></li><li><p><span><a href="#13">EFFECT OF PUTRESCINE ON CORN (ZEA MAYS) TOLERANCE TO SALINITY STRESS</a></span><br><span>Anuja  Walke, Meghan  Richline (Sophomore)</span></p></li><li><p><span><a href="#14">EFFECTS OF INHIBITING C-SRC ON THE DIFFERENTIATION OF RAT BONE MARROW AND MC3T3 OSTEOBLASTS</a></span><br><span>Ashley Dinkel (Senior, Biology)</span></p></li><li><p><span><a href="#15">EXPRESSION OF SIT IN THE MC3T3 PREOSTEOBLASTIC CELL LINE DURING DIFFERENTIATION</a></span><br><span>Richard Pasquali (Senior, Biology)</span></p></li><li><p><span><a href="#16">FINDING ACTIVE SITES IN PROTEINS USING PERL</a></span><br><span>Bryan Delfing (Junior, Bioinformatics)</span></p></li><li><p><span><a href="#17">FLUORESCENCE CHARACTERIZATION OF THE INDIVIDUAL COMPONENTS OF A SOLVATOCHROMIC CROSS MEMBRANE PROBE</a></span><br><span>Ethan Vallellanes (Senior, Biochemistry), Aleksandar Goranov (Junior, Chemistry), Alexa Dy (Graduated, Biochemistry)</span></p></li><li><p><span><a href="#18">INVESTIGATING THE ROLE OF CPAR-1 IN CELL DIVISION IN THE NEMATODE C. ELEGANS. </a></span><br><span>Madeleine Maas (Junior, Biology), Jenna-Marie Tracy (Senior, Biology)</span></p></li><li><p><span><a href="#19">INVESTIGATION OF MAGNETO-ELASTIC COUPLING IN AMORPHOUS MAGNETIC ALLOYS</a></span><br><span>Kamil Nowak (Junior, Engineering Physics), Christopher Bolton (Senior, Engineering Physics), Patrick Rehain (Junior, Physics), Nathaniel Birrer (Junior, Engineering Physics)</span></p></li><li><p><span><a href="#20">KI67:  A MARKER OF CELL PROLIFERATION FOR USE IN OSTEOBLAST CULTURES</a></span><br><span>Brandon Lui (Senior, Biology)</span></p></li><li><p><span><a href="#21">LIVE IMAGING AND ANALYSIS OF WILD-TYPE VERSUS CENP-A DEPLETED EMBRYOS IN THE NEMATODE, CAENORHABDITIS ELEGANS</a></span><br><span>Meghann Herman (Sophomore, Biology), Brianna Romer (Junior, Biology)</span></p></li><li><p><span><a href="#22">OLEFIN METATHESIS IN THE UNDERGRADUATE ORGANIC LABORATORY</a></span><br><span>Thanuka Udumulla Arachchilage (Junior, Chemistry), Amran Hussain (Junior, Biochemistry)</span></p></li><li><p><span><a href="#23">OPTIMAL NITROGEN AND LIGHT REGIMES TO GROW A MEDICINAL HERB, CALENDULA OFFICINALIS</a></span><br><span>Safal Shrestha (Junior, Bioinformatics)</span></p></li><li><p><span><a href="#24">ORGAN-SPECIFIC AND LIGHT EFFECT ON THE ACCUMULATION OF TOTAL PHENOLIC COMPOUNDS IN ARTEMESIA ANNUA</a></span><br><span>Nicholette  Ivezaj (Junior, Biology), Richard Apramian (Junior, Biology)</span></p></li><li><p><span><a href="#25">OVER-EXPRESSION AND PURIFICATION OF E. COLI RECOMBINATION PROTEIN RECR AND ITS INTERLOCKING MUTANTS USING VARIOUS PROTEIN CHROMATOGRAPHIC METHODS</a></span><br><span>Soo Jin Koo (Junior, Biochemistry), Christine Jaipersaud (Sophomore, Biochemistry), Jennifer Trabucco (Sophomore, Biochemistry)</span></p></li><li><p><span><a href="#26">PGE1 VS PGE2 IN THE PROMOTION OF MSC DIFFERENTIATION TO OSTEOBLASTS</a></span><br><span>Sophie Lazar (Senior, Biology), Alisha  Malla (Senior, Biology), Dean  Koenig (Senior, Biology)</span></p></li><li><p><span><a href="#27">QUANTITATIVE MEASURE OF VASCULAR SMOOTH MUSCLE RELAXATION IN PRIMARY RAT DESCENDING AORTA CELLS</a></span><br><span>Candace  Morales-Wilde (Junior, Biology/Chemistry), Brianna  Brianna Dziedzic (Senior , Biology )</span></p></li><li><p><span><a href="#28">SILENCING THE KINASE PROTEIN C-YES</a></span><br><span>Chloe Veras (Junior, Biology)</span></p></li><li><p><span><a href="#29">SOLVENT-FREE SYNTHESIS OF CHALCONES AS ANTIOXIDANTS</a></span><br><span>Joseph Szorentini (Senior, Biochemistry), Matthew Von Bargen (Senior, Biochemistry), Daniel Ferrante (Junior, Chemistry)</span></p></li><li><p><span><a href="#30">SYNTHESIS OF HYDRAZONES AS ANTIBIOTICS </a></span><br><span>Ryan Ludvigsen (Senior, Biochemistry), Chris  Warren (Senior, Psychology), Sydney  Kauffman (Sophomore, Biology), Karina Borda (Sophomore, Biology), Jennifer Trabucco (Sophomore, Biochemistry)</span></p></li><li><p><span><a href="#31">TESTING THE EFFICACY OF RNAI FEEDING IN C. ELEGANS</a></span><br><span>Matthew Von Bargen (Senior, Biochemistry), Samantha Davis (Junior, Biology)</span></p></li><li><p><span><a href="#32">TESTING THE EFFICACY OF RNAI INJECTION IN C. ELEGANS</a></span><br><span>JJ Fritsch (Junior, Biology), Austin Marcelo (Sophomore, Biology)</span></p></li><li><p><span><a href="#33">TRANSCRIPTOME WIDE ANALYSIS OF CIS-REGULATORY G-QUADRUPLEX MOTIFS AND CSTF BINDING SITES INVOLVED IN MAMMALIAN POLYADENYLATION</a></span><br><span>Haley Wight (Senior, Bioinformatics)</span></p></li><li><p><span><a href="#34">TRANSCRIPTOME WIDE DISTRIBUTION PATTERNS OF CONSERVED CIS-REGULATORY U-RICH SEQUENCE MOTIFS IN THE HUMAN POLYA REGIONS</a></span><br><span>Mingzhao Liu (Senior, Bioinformatics &amp; Computer Science)</span></p></li><li><p><span><a href="#35">TREE MORTALITY, CANOPY OPENINGS AND PLANT SPECIES COMPOSITION IN A HARDWOOD FOREST OF THE NEW JERSEY HIGHLANDS</a></span><br><span>Michael Finocchio (Senior, Environmental Science), Stuart Brown (Senior, Environmental Science), Ashley Olsen (Senior, Environmental Science)</span></p></li><li><p><span><a href="#36">TWO FALL SEASONS OF RAPTOR MIGRATION DATA FROM THE RAMAPO MOUNTAINS: HELPING UNDERSTAND REGIONAL TRENDS</a></span><br><span>Zachary Batren (Senior, Environmental Science), Patricia Goense (Senior, Environmental Science), Dylan Keastead (Senior, Environmental Science), Lawrence Kelsey (Senior, Environmental Science), Lichtenberger Courtney (Senior, Environmental Science)</span></p></li></ul></div><h2 class="full">Abstracts (Full Text)</h2><div class="abstract"><a name="0"></a><h3>A NATIVE WILDFLOWER MEADOW AT RAMAPO COLLEGE OF NEW JERSEY: ESTABLISHMENT, MAINTENANCE AND DATA FROM THE FIRST TWO YEARS</h3><p><b>Author(s):  </b><span>Melissa Mayberry (Senior, Bioinformatics &amp; Computer Science)</span></p><p><b>Advisor(s):  </b><span>Eric Wiener</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>Native wildflower meadows provide rich food resources for insect pollinators, and are increasingly being recognized as a sustainable option for the management of open areas. The main objectives of this study were to: a) use organic methods to establish a 1/4 acre native wildflower meadow at Ramapo College, b) monitor plant establishment and use of the meadow by insect pollinators. Turf was removed, and a light addition of organic mulch was rototilled into the soil. Seeds of 20 native wildflower species were spread over the area, and 325 plugs of 4 species were planted. Maintenance consisted of occasional watering and weeding of some non-native plant species during the first year, and occasional weeding and no watering the second year. Surveys of 25 plots (1 m2 each) revealed 10.2 native wildflower plants / m2 during the first year, and 19.9 native wildflower plants / m2 during the second year. Chamaecrista fasciculata, which was the only annual species planted, flowered during the first year. Flowers of many species were extremely abundant during the second year. The most common species were <i>Agastache foeniculum, Asclepias tuberosa, Coreopsis lanceolata, Monarda fistulosa, Solidago speciosa, Symphyotrichum novae-angliae, Echinacea purpurea</i>. Thirteen other planted species flowered, and plants of 8 additional native wildflower species established in the meadow naturally. Roaming surveys revealed extraordinarily high abundance of pollinators during the second year. A total of 24 species of native <i>Hymenoptera, Lepidoptera and Diptera</i> were observed. <i>Apis mellifera</i> rarely foraged in the meadow during spring and summer. However, they foraged intensively on goldenrods and asters in early autumn. Overall, the first two years of this study suggest that establishing native wildflower meadows can be a highly sustainable and aesthetically pleasing landscaping alternative, while providing forage for insect pollinators, and excellent opportunities for research and public education.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="1"></a><h3>AC-MAGNETIC SUSCEPTIBILITY OF AMORPHOUS MAGNETIC ALLOYS</h3><p><b>Author(s):  </b><span>Nathaniel Birrer (Junior, Engineering Physics), Patrick Rehain (Junior, Physics), Kamil Nowak (Junior, Engineering Physics), Christopher Bolton (Senior, Engineering Physics)</span></p><p><b>Advisor(s):  </b><span>Catalin Martin</span></p><p class="type"><span>Presented in Session 1 (no poster)</span></p><p class="abstract_text"><p>We have designed and built a magnetic susceptometer based on mutual inductance technique. The set-up was used to measure the ac-magnetic susceptibility of amorphous magnetic alloys and to investigate their magneto-elastic properties. We performed experiments both on bulk magnetic strips and on thin films deposited on silicon substrate. The data was analyzed based on a model developed to account for the strain induced in the amorphous alloy under the action magnetic field. We found clear, sharp signature of both resonance and anti-resonance, consistent with the theoretical model. Particularly important is the study of thin films, which opens the possibility for miniaturized magnetic and/or strain sensors. </p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="2"></a><h3>ANALYSIS OF SIT EXPRESSION ON A PROTEIN BASED LEVEL</h3><p><b>Author(s):  </b><span>Taylor  Donnelly (Junior, Biology)</span></p><p><b>Advisor(s):  </b><span>Thomas Owen</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>In studies performed with our collaborators at Temple Medical School, approximately 150 genes potentially involved in the regulation of bone mass in rats were isolated.  One of these genes is SIT, a member of a family of transmembrane adapter proteins which recruit other signaling proteins to the cell membrane. Although the published literature about SIT only reports its expression and function in T-cells, recent evidence from our lab demonstrated that SIT is expressed in osteoblasts at the mRNA level and more importantly, that the bones of SIT knock-out mice show dramatic decreases in the amount of trabecular bone. The work in the study presented here is aimed at establishing conditions through which to identify SIT protein in rat cell and tissue samples using Western blot analysis.  To begin this work, total cellular protein was prepared from ROS 17/2.8 rat osteosarcoma cells.  This protein sample was run on SDS-PAGE and blotted to a PVDF membrane.  Both BSA and milk were tried as blocking agents and milk proved superior.  Various dilutions of a commercially available goat anti-SIT antibody were then tried along with various dilutions of a horseradish peroxidase-conjugated donkey anti-goat secondary antibody.  The signal was visualized using Clarity substrate and chemiluminescent imaging.  Subsequently, total protein samples were prepared from rat stomach, liver, heart, spleen and kidney and analyzed for SIT expression using the optimized conditions.  Interestingly, while a strong SIT signal is visible in the ROS cells with a 10 sec exposure, SIT could only be detected in rat kidney tissue and then only after an extended period of exposure (15 minutes).  Since SIT is a transmembrane protein, we are planning on preparing membranes from various rat tissues in order to enhance any signal and are also planning on exploring whether the high abundance of SIT protein in ROS cells is unique to them or might be a consequence of their tumor origin.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="3"></a><h3>ANDROID APPLICATION TO AID BIRD IDENTIFICATION IN THE FIELD</h3><p><b>Author(s):  </b><span>Thomas Harden (Senior, Biology)</span></p><p><b>Advisor(s):  </b><span>Eric Wiener, Amruth Kumar</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>A user-friendly Android application is being designed to enable bird identification for users of all skill levels, but is primarily directed towards novice birdwatchers. The overall approach of the app is to guide users through an identification key that generates images and detailed information of likely species via a search module where users systematically select parameters based on the traits of any bird observed in the field. The user inputs the date and location of the bird, and then selects from lists of choices for two or more of the following variables: body shape and size, colors of different body parts, pattern of wing beats, flocking behavior, habitat, nesting information, and more. In terms of implementation, the feature data will be saved in a database on a server. An Android app will retrieve and use this data to assist the user. A longer-term goal is to allow users to save entries into personalized, online databases, as well as make modifications to enhance the software. Overall, the app is expected to serve as an excellent educational tool, and may help empower citizen science and even environmental professionals.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="4"></a><h3>ANTIBIOTIC RESISTANCE AMONG ISOLATES OF STAPHYLOCOCCUS AUREUS FROM RAMAPO COLLEGE STUDENTS</h3><p><b>Author(s):  </b><span>David Cifelli (Sophomore, Biology), Sophia Penkrat, Amber Smith</span></p><p><b>Advisor(s):  </b><span>Thomas Owen</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>According to published estimates, up to one-third of the population carry the Gram-positive Staphylococcus aureus in their nose. Although the presence of this bacterial species is usually clinically silent, it can manifest into illness in certain individuals. Increasingly, drugs such as methicillin and vancomycin are not effective in treating S. aureus infection, with approximately 2% of the population carrying methicillin-resistant Staphylococcus aureus (MRSA). The purpose of this study was to analyze the sensitivity of S. aureus, collected from a group of Ramapo College students (~18-25 years old), to eight antibiotics often used in treating S. aureus infections, using the Kirby-Bauer disc diffusion test. These antibiotics included ampicillin, ciprofloxacin, clindamycin, linezolid, minocycline, oxacillin, vancomycin, and sulfamethoxazole/trimethoprim. S. aureus obtained from the nasopharyngeal cavity using self-obtained nasal swabs was plated immediately onto mannitol salt agar (MSA) plates. Potential S. aureus colonies (MSA growth positive and mannitol fermentation positive) were verified as S. aureus by positive coagulase test. These isolates were then subjected to Kirby-Bauer testing in triplicate. It was hypothesized that the study would return data consistent with the estimate of S. aureus being present in about one-third of the population. It was further postulated that some isolates would be resistant to other antibiotics, indicating the possible presence of MRSA, vancomycin-resistant Staphylococcus aureus (VRSA), or other drug-resistant variants. The most recent data collected indicates that, among the 233 Ramapo College students sampled, the presence of S. aureus is 19%, and that 5.5% of the students sampled harbor S. aureus, which is resistant to one or more of the eight antibiotics.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="5"></a><h3>BACTERIAL GROWTH STUDIES OF GUT MICROBES INCLUDING LACTOBACILLUS RHAMNOSUS GG AND ESCHERICHIA COLI HS USING UV-VIS SPECTROPHOTOMETRY AND QUANTITATIVE PCR (QPCR)</h3><p><b>Author(s):  </b><span>Devashri Parikh (Sophomore, Biochemistry), Pritha Aggarwal (Sophomore, Biochemistry), Katerina Djambazova (Sophomore, Biochemistry, Psychology)</span></p><p><b>Advisor(s):  </b><span>Seung-Sup Kim</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>The long term goal of this project is to understand the symbiosis among gut microbes and the current goal of the project is to develop a quantitative analysis method to measure the growth of specific gut microbes including Lactobacillus rhamnosus GG and Escherichia coli HS. In this paper, we discussed the early stage of our development including the growth study of the target bacteria using UV-VIS spectrophotometry and quantitative PCR analysis. The specific strains of the two target bacteria in use have been identified using common 16S rRNA sequencing. The growth studies of both bacteria were completed under aerobic and anaerobic conditions. For the UV-VIS spectrophotometry analysis, an OD600 measurement was used and the result clearly showed the differences in the growth phases of the two different bacteria under the same conditions (Tryptic soy broth media, 37°C and aerobic or anaerobic). For quantitative PCR analysis, a 16S rRNA template was employed for the PCR reactions and the results showed similar outcomes with the growth studies using the UV-VIS spectrophotometry analysis. Currently, we are trying to connect the two different quantitative bacteria growth analysis techniques including cell mass analysis using: (1) UV-VIS spectrophotometry and (2) quantification of strain specific 16S rRNA using quantitative PCR.  In the future, this combination of the two different methods may provide us one of the possible tools to quantitate a specific bacteria strain from a mixture of different microbes.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="6"></a><h3>COMPARATIVE ANALYSIS OF PHASEOLUS VULGARIS AND CICER ARIENTIUM CONVENTIONALLY VERSUS VERTICAL HYDROPONICALLY GROWN</h3><p><b>Author(s):  </b><span>Brenda Arthur (Junior, Biology), Stephanie  Morfi (Senior, Biology)</span></p><p><b>Advisor(s):  </b><span>Yan Xu</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>Vertical farming in urban environments has sprung out of a need to find alternatives to common practices in industrial agriculture. The methods used for industrial agriculture are inefficient and have had a widespread negative impact on the environment, mainly because of the excess water used in farming. Hydroponics refers to a new form of production where plants are grown in a nutrient-enriched water solution with a growing medium (Rockwool), rather than in soil. These materials are in containers that are stacked at a 45-degree angle in a vertical hydroponics apparatus. There are a number of environmental and economical benefits associated with hydroponic production versus traditional soil-based agricultural production. Hydroponic farming has shown to produce plants at a faster rate that ripen earlier and produce up to ten times the yield than that of soil-grown plants. These plants are also said to have a greater nutritional value. Hydroponics also uses 80% less water than bench top, flat growth pattern farming. This three-month study aims to validate these claims by using <i>Phaseolus vulgaris</i> and <i>Cicer arientium</i>. Both bean species are planted vertically as well as on a bench top in a Ramapo College campus greenhouse. Photos will be taken and water will be measured periodically for a three-month period for analysis.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="7"></a><h3>COMPARING FOOD KNOWLEDGE AND DIET HABIT OF COLLEGE STUDENTS IN US AND CHINAS METROPOLITAN AREAS</h3><p><b>Author(s):  </b><span>Brenda Arthur (Junior, Biology), Stephanie  Morfi (Senior, Biology)</span></p><p><b>Advisor(s):  </b><span>Yan Xu</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>Diet related diseases such as cardiovascular disease and cancer are consistently among the leading causes of death (American Cancer Society). Many college students do not meet the minimum recommended intake of dietary fiber, fruits, and vegetables, while they exceed recommended intakes of total fat and saturated fat, sugar, and sodium. The National Eating Disorders Association notes that the average college freshman gains about 2.5 to 3.5 lbs. during the first year on campus. 25-32% of college students may battle an eating disorder. College students are at a time and place in their lives where their behavior is conductive to change. Most students gain the freedom to choose what to eat since entering college. College campus may serve as a crucial setting to overcome perceived barriers to healthy diet habits, and implement effective interventions. This college transition is a good time to examine what students are learning about foods and how their diet is affected because of it. The sooner college students form a good diet habit, the larger the impact they can make on their own healthcare, family, neighborhood and society. Many social and economic factors may influence people’s decisions on choosing foods. We were curious about how the current trend of food knowledge was accepted in a different society. This study compares students living in the New York City (United States of America) and Shanghai (China) metropolitan areas. The pilot study was done in Spring 2015 comparing Ramapo College of New Jersey (RCNJ) students to Shanghai Normal University (SHNU) students.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="8"></a><h3>DEVELOPMENT OF A BIOINFORMATICS APPLICATION FOR TRANSCRIPTOME WIDE GENE FUNCTION ANALYSIS OF MRNAS WITH MAPPED CIS-REGULATORY G-QUADRUPLEX MOTIFS.</h3><p><b>Author(s):  </b><span>Melissa Mayberry (Senior, Bioinformatics &amp; Computer Science)</span></p><p><b>Advisor(s):  </b><span>Scott Frees, Paramjeet Bagga</span></p><p class="type"><span>Presented in Session 2 and as a Poster</span></p><p class="abstract_text"><p>G-quadruplexes are three-dimensional,  guanine rich structures with significant roles in the post-transcriptional cis-regulation of mRNA expression. Their formation in the mRNA's 3' untranslated region (UTR), 5'-UTR, or in the vicinity of a polyadenylation site can influence the expression of that gene. Therefore, G-quadruplexes are involved in certain diseases, such as cancer, making them important therapeutic targets.</p><p>The purpose of this project was to develop an application which could categorize a large number of mRNAs by their biological processes, molecular functions, and cellular component location data, and then determine the prevalence of mRNAs with cis-regulatory G-quadruplex motifs in the translated, untranslated, and polyA regions of these clusters.</p><p>The bioinformatics application, QGRS Ontology, that we have developed allows for a variety of input for G-quadruplex characteristics and subsets of thousands of mRNAs with gene ontology data in our database to cater to a variety of future research projects. QGRS Ontology can be used to analyze which gene ontology terms are expressed in a list of mRNAs of interest, and, of these terms, which have a notably high or low prevalence of mRNAs with G-quadruplexes. The data generated by this application identifies important mRNAs, and relationships between G-quadruplexes and other motifs which may warrant additional research.</p><p>We've used QGRS Ontology on our dataset of mRNAs containing CStF binding sites, which assist in polyadenylation. We observed that mRNAs containing weak CStF binding sites that are associated with cis-regulatory G-quadruplex motifs downstream of the polyA site are involved in positive regulation of cell proliferation (30.3%) and transmembrane transport (27.2%). These data suggest that G-quadruplexes may be more necessary to support the weak CStF binding site in the mRNAs involved in these processes than in those involved in processes such as transcription from RNA polymerase II promoter (13.3%).</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="9"></a><h3>DEVELOPMENT OF CAPABILITY FOR PRODUCTION OF MONOCLONAL ANTIBODIES</h3><p><b>Author(s):  </b><span>Jennifer Howard (Junior, Biology)</span></p><p><b>Advisor(s):  </b><span>Thomas Owen, Joost Monen</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>In many techniques used to identify proteins protein expression, interactions, and cellular and tissue expression, such as Western Blot and immunohistochemistry, it is necessary to have a high quality antibody which can recognize the desired protein.  Many proteins do not have antibodies commercially available, and if they are available they can be very expensive.  In addition, some of the commercially available antibodies are not of very high quality.  The goal of this project is to develop the technical capabilities to eventually be able to produce our own monoclonal antibodies to desired proteins, including bone proteins and C. elegans proteins.  We are piloting these techniques in order to eventually produce an antibody to the intracellular portion of rat SIT, a protein which we believe to be important in osteoblast cell signaling.  The intracellular portion of SIT is only about 120 amino acids in length, which makes it a good first candidate protein due to its medium length.  Our current work is focused on expressing this protein fragment in  E. coli and the purifying it.  Eventually, the purified protein fragment will be injected into mice in order to evoke an immune response.  B cells from these mice will then be fused with hybridoma cells in order to produce monoclonal antibodies for laboratory use.  </p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="10"></a><h3>DEVELOPMENT OF LENTIVIRAL DELIVERY OF OVEREXPRESSION AND SILENCING VECTORS TO PRIMARY CELLLS</h3><p><b>Author(s):  </b><span>Kate  Reilly (Senior, Biology), Sara Luby (Senior, Biology)</span></p><p><b>Advisor(s):  </b><span>Thomas Owen</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>A previous study with our collaborators at Temple University School of Medicine, identified SIT, a member of a family of transmembrane adapter proteins (TRAPs), as a gene potentially involved in regulating bone mass. Little is known about this gene with the only publications describing its role in the immune system. Recent evidence from our lab shows that SIT is expressed in osteoblasts at the mRNA level and more importantly; that the bones of SIT knock-out mice show dramatic decreases in the amount of trabecular bone present. Our current work is aimed at both overexpressing and silencing SIT in cultured cells in order to begin to establish its function in bone. Introducing DNA into primary osteoblasts or stem cells by conventional means is problematic. One method to more efficiently deliver DNA is to use a Lentivirus vector. Genetically modified, nonreplicating virus components are commercially available for both overexpressing and silencing genes. In order to optimize the production of Lentiviral vectors using this system we focused on the overexpression of B-galactosidase in order to have an easily assayed target.  Production of these viral vectors under appropriate biosafety conditions require the transfection of four plasmids simultaneously into the 293TN packaging cell line. We first optimized the use of a new transfection reagent for these cells and then optimized the ratios of the four plasmids for the best viral production as determined by staining cells which were infected by our virus. We are currently producing a virus which contains shRNA to silence B-galactosidase and also developing methods to concentrate the viral particles in order to proceed to work on the primary cells.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="11"></a><h3>DEVELOPMENT OF SILENCING TOOLS FOR SRC PROTOC-ONGOGENE, NON RECEPTOR TRYOSINE GENE (C-SRC) AND SIGNALING THRESHOLD REGULATING TRANSMEMBRANE ADAPTOR 1 GENE (SIT1)</h3><p><b>Author(s):  </b><span>Nicole Nebbia (Senior, Biology), Donna Romhen (Senior, Biology), Emma Murray (, )</span></p><p><b>Advisor(s):  </b><span>Thomas Owen, Nicole Rodstrom</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>c-src is a non-receptor tyrosine kinase which is part of a gene family that is responsible for many biological functions including regulation of embryonic development and cell growth.  c-src is known to also play a role in bone since when the gene is knocked-out in mice, the result is osteopetrosis (too much bone), resulting from underactive osteoclasts and overactive osteoblasts.  However, the molecular mechanisms underlying the over activity of osteoblasts are unclear.  A potential target protein which may be phosphorylated by c-src in osteoblasts is SIT (SHP2-interacting transmembrane adaptor).  Previous work has demonstrated that SIT is somehow involved in regulation of bone mass since when the SIT gene is knocked-out, lower bone mass results.  The intracellular domain of SIT has 4 tyrosines which, in the immune system, are known to be phosphorylated by other kinases in the c-src family.  The goal of our work is to develop molecular reagents through which c-src and SIT can be specifically silenced so that we will be able further assess the roles of both in bone as well as address the question of whether the two proteins interact.  We have chosen to develop short hairpin RNAs (shRNAs) for each gene and to clone them into a lentivirus transfer vector for functional analysis.  Online software was used to choose two sets of potential shRNAs for each gene.  Oligonucleotides based on these sequences were designed with the appropriate restriction enzyme sites needed for cloning and with a loop sequence to separate the two sides of the hairpin.  The oligos were phosphorylated and the matching top and bottom strands annealed.  They were then ligated into pSIH1, which had been digested with BamH1 and EcoR1.  Once ligated, we transformed them into E. coli which were plated on LB-ampicillin plates.  Clones were identified by restriction enzyme digestion and confirmed by DNA sequencing.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="12"></a><h3>DOES CHRONIC EXPOSURE TO TRICLOSAN LEAD TO RESISTANCE TO UNRELATED ANTIBIOTICS?</h3><p><b>Author(s):  </b><span>Alexandria Farrell (Senior, Biology), Hailey Ebue (Junior, Biology)</span></p><p><b>Advisor(s):  </b><span>Thomas Owen</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>Triclosan is an antibacterial and antifungal agent that is found in personal care products such as toothpaste, soaps, and deodorants. Triclosan was used to reduce bacterial contamination of various surfaces, but there is now concern raised about using the drug. Triclosan resistance in bacteria has been linked antibiotic resistance. The purpose of this research is to study whether bacteria can build a triclosan resistance, and whether or not a gradual exposure to the drug can in fact lead to resistance against other important antibiotics. Oral bacteria were obtained by swabbing the inner cheek. Each person obtained a Gram positive bacterial species which was then used in our studies. To determine the initial susceptibility of the bacteria to antibiotics, a Kirby-Bauer test was done to create a baseline susceptibility to nine different antibiotics. Then we determined the minimal inhibitory concentration (MIC) for triclosan for both bacterial species.  After finding the MIC, each bacterial species was inoculated into 5 mL of tryptic soy broth (TSB) containing half the concentration of the MIC. Every other day cells were then subcultured into 5 mL of TSB and the concentration of the triclosan was increased by 10%. Until the end of the semester we will continue this increase in exposure to triclosan and run a final Kirby-Bauer disc test using the same antibiotics, to determine if there has been a change in antibiotic susceptibility in either of our bacterial species.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="13"></a><h3>EFFECT OF PUTRESCINE ON CORN (ZEA MAYS) TOLERANCE TO SALINITY STRESS</h3><p><b>Author(s):  </b><span>Anuja  Walke, Meghan  Richline (Sophomore)</span></p><p><b>Advisor(s):  </b><span>Yan Xu</span></p><p class="type"><span>Presented in Session 1 and as a Poster</span></p><p class="abstract_text"><p>Corn (Zea mays), along with rice (Oryza sativa) and wheat (Triticum spp.), is the most valued cereal crop in the world. It can be used as food substance, animal feed, and alternative fuel. However, the yield of this cereal crop is often restricted by environmental pressures such as salinity stress. Globally, salt stress limits agricultural productivity on 20% of the cultivated land and half of the irrigated land worldwide. It would be highly desirable if a management strategy could be developed for Z. mays to cope with salinity. Putrescine, a polyamine, has been observed to increase the resistance of other plants to salt stress. In this study, Z. mays plants were treated exogenously with putrescine before and during 400mM salt treatments in order to determine the chemical’s effect on the tolerance of Z. mays to salinity stress. With treatments of 50 mg/L and 150 mg/L putrescine, it was observed that both treatments inhibited the Z. mays plants’ development more than the controls, which were given the salt solution and no putrescine treatment. It was observed that leaf length and number of leaves on the plants decreased with application of putrescine in comparison to the control. Likewise, the putrescine treated plants photosynthesized less in comparison to the control. However, though both treatments retarded the growth of the plants, those treated with 50 mg/L putrescine had a slower growth rate than those treated with 150 mg/L putrescine. This observation warrants further study on the effect that higher concentrations of putrescine have on Z. mays. Exogenous putrescine application was also noticed to prevent salinity uptake through the soil. The moisture content in the soil of the treated plants was elevated in comparison to the control, with 50 mg/L treated plants absorbing the least amount of salt treatment. Further studies are currently being conducted on the effect of putrescine on related gene expression. </p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="14"></a><h3>EFFECTS OF INHIBITING C-SRC ON THE DIFFERENTIATION OF RAT BONE MARROW AND MC3T3 OSTEOBLASTS</h3><p><b>Author(s):  </b><span>Ashley Dinkel (Senior, Biology)</span></p><p><b>Advisor(s):  </b><span>Thomas Owen</span></p><p class="type"><span>Presented in Session 1 and as a Poster</span></p><p class="abstract_text"><p>To maintain proper bone density in the body, the activity of bone forming osteoblasts and bone resorbing osteoclasts must be balanced. C-src kinase is an enzyme known to play an important role in the functioning of both of these cell types. When the c-src gene is deleted in mice, there is a decrease in osteoclast activity and an increase in the rate of differentiation of osteoblasts. This effect on bone formation can be shown in vitro using cultures of normal osteoblasts isolated from rat calvaria which undergo a series of developmental steps ultimately resulting in the formation of multilayered nodules of cells with a structure similar to bone. When c-src kinase is inhibited in these cells using the inhibitor compound PP2, their differentiation increases. Prior to the work reported here, other students attempted to study this mechanism by inhibiting c-src in rat-derived Ros 17/2.8 osteosarcoma cells using PP2 as the active inhibitor and PP3 as a non-active control. When c-src was inhibited in these tumor-derived cells, markers of osteoblastic differentiation were decreased, not increased as in calvarial cells. Our goal was to determine whether treatment timing, tissue of origin, or another property of the cells used caused this discrepancy in results. We used a primary rat bone marrow osteoblast differentiation system and an MC3T3 osteoblast differentiation system to explore the effects of c-src inhibition in multiple models. When treated with either PP2 or PP3, the primary rat bone marrow cells demonstrated a higher level of osteoblastic differentiation upon treatment with the inactive inhibitor PP3, while the MC3T3 cells produced a greater quantity of mineralized bone when treated with the active inhibitor PP2. This suggests that increased osteoblast differentiation following PP2 treatment is not universally absent in immortalized cell lines and that this effect may be specific to osteoblasts which produce intramembranous bone as opposed to endochondrial bone.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="15"></a><h3>EXPRESSION OF SIT IN THE MC3T3 PREOSTEOBLASTIC CELL LINE DURING DIFFERENTIATION</h3><p><b>Author(s):  </b><span>Richard Pasquali (Senior, Biology)</span></p><p><b>Advisor(s):  </b><span>Thomas Owen</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>Bone is quite metabolically active, with bone formation and resorption constantly occurring to maintain the skeleton’s integrity. While tremendous progress in understanding the molecular signaling pathways underlying bone metabolism has been made over the past few decades, new aspects of signaling are continually being identified. MC3T3 is a mouse preosteoblastic cell line, which can differentiate in culture into mineralized osteoblasts over about a three-week period.  We are using this cell line in order to first study the expression and ultimately the function of a novel signaling protein, SIT, in osteoblast differentiation. In order to monitor the growth and differentiation of the cells, the expression of two bone phenotype marker genes, alkaline phosphatase (AP) and osteocalcin (OC) will be assessed over time.  AP expression increases early in the differentiation process and OC increases later in association with the deposition of mineral onto the collagenous bone matrix.  The expression of the SIT gene will subsequently be assessed.  For all three genes, gene expression on the RNA level will be determined using quantitative RT-PCR (qRT-PCR) and normalized to expression of the housekeeping gene GAPDH.  PCR primer sets were chosen for each of the three bone genes, as well as for GAPDH, and optimal conditions for amplification was determined for each set using a representative MC3T3 RNA sample which was reverse transcribed into cDNA.  MC3T3 cells are currently being grown under differentiation conditions and samples will be harvested for RNA isolation at various intervals.  The expression of AP, OC, and SIT RNA will then analyzed in cDNA made from these samples and SIT expression will be compared to that of the known bone phenotype marker genes.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="16"></a><h3>FINDING ACTIVE SITES IN PROTEINS USING PERL</h3><p><b>Author(s):  </b><span>Bryan Delfing (Junior, Bioinformatics)</span></p><p><b>Advisor(s):  </b><span>Ashley Stuart</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>Perl scripting was used to write a program that generates an image in PyMOL that displays the active site(s) in a protein structure. PyMol is a molecular graphics program that can be used to visualize biological molecules in 3D. Known active sites or ligand binding sites in protein structures often define the biological importance of that protein family.  Active site templates from known protein structures were built that represent the active site amino acids in 3 dimensional space, and these templates will be used to look for new active sites in protein models by looking for a  3 dimensional pattern similar to those found in the previously identified active sites. This will allow the generation of protein images that contain these binding sites in PyMol. As part of the project, a web-based tutorial  was developed to teach others how to use PyMOL to graphically represent functional information about proteins.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="17"></a><h3>FLUORESCENCE CHARACTERIZATION OF THE INDIVIDUAL COMPONENTS OF A SOLVATOCHROMIC CROSS MEMBRANE PROBE</h3><p><b>Author(s):  </b><span>Ethan Vallellanes (Senior, Biochemistry), Aleksandar Goranov (Junior, Chemistry), Alexa Dy (Graduated, Biochemistry)</span></p><p><b>Advisor(s):  </b><span>Lorraine Tan</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>Fluorescence spectroscopy was used to characterize solvent-sensitive dual-fluorescent molecules in solvents of varying polarities.  The fluorescent probe molecules utilized in this study were Coumarin-120, a reactive Dansyl Chloride, and a nonreactive Capped Dansyl.  There were no significant trends found between solvent polarity and emission maxima of any probe.  Based on these results, the study shifted to using an Ethanol-Water combination in attempts to modify the polarity of the solutions while maintaining the system composition. Fluorescence emission of each probe was dependent on its solubility in the ethanol-water solvent system.  Linearity was achieved for all three probes when ethanol was >60%.  In theory, two probes can be tethered together with a carbon chain to produce a cross-membrane probe which can report on the environment by self orientation across the membrane a reverse micelle.  There should be no discrepancy between the individual components and the cross membrane probe when analyzing the fluorescent trends which will be determined when the cross membrane probe is subjected to these experiments</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="18"></a><h3>INVESTIGATING THE ROLE OF CPAR-1 IN CELL DIVISION IN THE NEMATODE C. ELEGANS. </h3><p><b>Author(s):  </b><span>Madeleine Maas (Junior, Biology), Jenna-Marie Tracy (Senior, Biology)</span></p><p><b>Advisor(s):  </b><span>Joost Monen</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="19"></a><h3>INVESTIGATION OF MAGNETO-ELASTIC COUPLING IN AMORPHOUS MAGNETIC ALLOYS</h3><p><b>Author(s):  </b><span>Kamil Nowak (Junior, Engineering Physics), Christopher Bolton (Senior, Engineering Physics), Patrick Rehain (Junior, Physics), Nathaniel Birrer (Junior, Engineering Physics)</span></p><p><b>Advisor(s):  </b><span>Catalin Martin</span></p><p class="type"><span>Presented in Session 2 (no poster)</span></p><p class="abstract_text"><p>Amorphous ferromagnetic alloys are increasingly attractive today for sensor applications, due to a large coupling between their magnetic and mechanical (elastic) properties. In general, the magneto-elastic coupling is probed by measuring the response of the alloy to a time or frequency varying magnetic field. Here we take the inverse approach, by vibrating the alloy mechanically and measuring the electromagnetic response. Using a comercially available vibrating reed, we designed a computer controlled experimental set-up for detecting the electromagnetic response under mechanical excitation. We find a sharp resonant response and calculate the elastic constant to be in good agreement with literature data. Through a systematic study of the response with applied external magnetic field, and with the amplitude of the mechanical excitation, we find the conditions for maximum sensitivity. Furthermore, we asses the potential of amorphous magnetic alloys for sensor applications, especially as devices for mechanical energy harvesting</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="20"></a><h3>KI67:  A MARKER OF CELL PROLIFERATION FOR USE IN OSTEOBLAST CULTURES</h3><p><b>Author(s):  </b><span>Brandon Lui (Senior, Biology)</span></p><p><b>Advisor(s):  </b><span>Thomas Owen</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>Osteoporosis is a disease that is characterized by a decrease in bone mass, typically occurring in older individuals.  This decrease results in an increased risk of fracture due to compromised bone structure and decreased mechanical strength.  Therapies to prevent further loss of bone are widely available but treatments to restore lost bone are limited. The overall goal of our continuing work is to discover novel genes involved in bone formation to better understand the function of the bone forming osteoblasts.  In previous studies by our collaborators at Temple University School of Medicine, 160 genes involved in the regulation of bone mass in rats were isolated.  One of these genes was subsequently identified as KI67.  KI67 is a protein whose expression is associated with cell proliferation and it is often used as a marker for tumor growth. In general, as cells undergo differentiation, proliferation decreases and so there is a decease in KI67 expression, so not surprisingly, this gene was identified as one with decreased expression in rats having increased bone mass.  Our research focused on establishing quantitative RT-PCR conditions through which the expression of KI67 could be assessed under various growth and differentiation conditions in osteoblast cell cultures. As a model, ROS17/2.8 rat osteosarcoma cells were plated and then harvested on days 3, 8, 11, and 14 after plating. RNA was isolated and then cDNA was generated through reverse transcription.  Conditions for PCR were then assessed using a gradient cycling profile to determine the optimal primer annealing temperature. qPCR was performed using KI67 primers and its expression was normalized to the expression of the housekeeping gene GAPDH.  Analysis of the data obtained from these experiments shows the expected decrease in KI67 expression in the ROS cells with time in culture and validates our assay for KI67 as a molecular marker of osteoblast proliferation.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="21"></a><h3>LIVE IMAGING AND ANALYSIS OF WILD-TYPE VERSUS CENP-A DEPLETED EMBRYOS IN THE NEMATODE, CAENORHABDITIS ELEGANS</h3><p><b>Author(s):  </b><span>Meghann Herman (Sophomore, Biology), Brianna Romer (Junior, Biology)</span></p><p><b>Advisor(s):  </b><span>Joost Monen</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>The centromeric protein CENP-A is critical for kinetochore deposition and chromosome segregation in mitosis.  In the nematode Caenorhabditis elegans two CENP-A homologs exist, HCP-3 and CPAR-1.  HCP-3 plays a conserved centromeric role critical for mitotic chromosome segregation, whereas CPAR-1's role has yet to be determined.  To assess whether CPAR-1 plays a complementary or perhaps a divergent role, this study aims to compare and contrast the functional consequences of RNAi depletion of HCP-3 or CPAR-1 in early embryogenesis.  Live imaging of GFP::Histone-H2b/GFP::a-tubulin expressing wild-type and HCP-3 depleted embryos was done on a Zeiss Axiovert 200M epi-fluorescent microscope at 30 second intervals.  Chromosome and spindle pole dynamics were analyzed and compared in single celled embryos revealing chromosome segregation failure and pre-mature spindle pole separation in HCP-3 depleted embryos.  These experiments are consistent with previous studies and provide a robust and reproducible assay to test RNAi depleted perturbations and analyze chromosome and spindle pole dynamics in live embryos.  Future experiments will include a similar assessment of CPAR-1 depleted embryos to characterize the functional role of CPAR-1 in early embryonic development.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="22"></a><h3>OLEFIN METATHESIS IN THE UNDERGRADUATE ORGANIC LABORATORY</h3><p><b>Author(s):  </b><span>Thanuka Udumulla Arachchilage (Junior, Chemistry), Amran Hussain (Junior, Biochemistry)</span></p><p><b>Advisor(s):  </b><span>Sarah Carberry</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>A procedure has been developed to introduce olefin metathesis in the undergraduate organic laboratory. The conversion of styrene to <i>trans</i>-stilbene using ruthenium-based catalysts was investigated to develop a suitable experiment that can be carried out safely, in a limited amount of time, and with simple characterization techniques such as TLC and melting point. The use of the Hoveyda-Grubbs catalyst under solventless conditions was determined to be the best procedure that resulted in pure <i>trans</i>-stilbene in good yields. Students encounter several green chemical principles and techniques in this procedure, including: use of less toxic solvents, use of effective catalysts, and solvent-free conditions. This laboratory procedure can be incorporated into the undergraduate lab sequence and provide the starting material for other established lab experiments, two of which will be discussed. </p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="23"></a><h3>OPTIMAL NITROGEN AND LIGHT REGIMES TO GROW A MEDICINAL HERB, CALENDULA OFFICINALIS</h3><p><b>Author(s):  </b><span>Safal Shrestha (Junior, Bioinformatics)</span></p><p><b>Advisor(s):  </b><span>Yan Xu</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>Alongside the use of <i>Calendula officinalis</i> (pot marigold) as an ornamental plant, it also has great pharmacological importance due to the cytotoxic, anti-inflammatory, and antiseptic properties of the flower extract. However, very little has been understood about the optimal indoor growth requirements of the plant. The aim of the project is to evaluate the responses of vegetative growth and flower production of Calendula under the interaction of differential nitrogen fertilizer and light intensity. Four treatments were imposed to grow Calendula plants in the greenhouse: 1) 100% light with nitrogen (in the form of 0.1 M ammonium nitrate); 2) 100% light without nitrogen; 3) 50% light with nitrogen; and 4) 50% light without nitrogen. Significant difference in vegetative growth was observed between plants treated with nitrogen under 100% light and 50% light, as shown by the height of the plant, the number of leaves, and the chlorophyll content in the leaves. Moreover, a higher number of flowers were produced on Calendula plants grown under 50% light without nitrogen. Currently we are analyzing frozen leaf, stem and flower samples with an aim to understand the molecular basis for above-mentioned morphological and physiological variations through differential gene expression analysis using qPCR. A repeated cycle of the treatments and measurements is also undergoing in the greenhouse to verify the findings and to collect more samples for further analysis.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="24"></a><h3>ORGAN-SPECIFIC AND LIGHT EFFECT ON THE ACCUMULATION OF TOTAL PHENOLIC COMPOUNDS IN ARTEMESIA ANNUA</h3><p><b>Author(s):  </b><span>Nicholette  Ivezaj (Junior, Biology), Richard Apramian (Junior, Biology)</span></p><p><b>Advisor(s):  </b><span>Yan Xu</span></p><p class="type"><span>Presented in Session 2 and as a Poster</span></p><p class="abstract_text"><p>Artemesia annua is a wormwood commonly found in temperate Asian climates; however, it is now grown in many non-indigenous regions around the world. Artemesia annua has been used as a medicinal herb for treating fevers, bacterial infections, autoimmune diseases, and upset stomachs. More recently, its medicinal compounds are being used for the treatment of malaria. Its characteristic fern-like leaves and yellow flowers produce many bioactive compounds. A common group of bioactive molecules are phenolic compounds. In this experiment, the total phenolic content (TPC) was first quantified in three different organs of the plant (leaves, stems, flowers). Secondly, Artemeisa plants were grown in three different light conditions (0% shade, 30% shade, and 50% shade) in the greenhouse on campus. Samples were taken at three different stages of its growth. The flowers, stems, and leaves of each plant were separated. Each sample was ground to a fine powder. The phenolic compounds were extracted with 50% methanol and then quantified using a colometric method. A calibration curve was generated with the absorbance readings of a gradient of Gallic Acid solutions. The spectrophotometer readings of each Artemesia annua extract was then taken to determine its TPC expressed as Gallic Acid Equivalents (GAE). The results are currently being analyzed.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="25"></a><h3>OVER-EXPRESSION AND PURIFICATION OF E. COLI RECOMBINATION PROTEIN RECR AND ITS INTERLOCKING MUTANTS USING VARIOUS PROTEIN CHROMATOGRAPHIC METHODS</h3><p><b>Author(s):  </b><span>Soo Jin Koo (Junior, Biochemistry), Christine Jaipersaud (Sophomore, Biochemistry), Jennifer Trabucco (Sophomore, Biochemistry)</span></p><p><b>Advisor(s):  </b><span>Seung-Sup Kim</span></p><p class="type"><span>Presented in Session 2 and as a Poster</span></p><p class="abstract_text"><p>The RecFOR pathway is one of the bacterial DNA recombination processes, where DNA double strand breaks are repaired by homologous recombination. While it is known that RecF, RecO and RecR proteins work together as a complex, mechanisms of the functions of the RecFOR complex have yet to be well understood. The long-term goal of this project is to study the structure-function relationship of E. coli RecR protein in the RecFOR pathway. The current goals of the project are to over-express and purify the RecR protein to study the oligomerization and the DNA-binding mechanisms of the RecR protein and its interlocking mutants, G39C and A172C, which were previously obtained in our group. In the first stage of the project, recombinant proteins of the native E. coli RecR and its mutant forms were produced in T7 bacterial over-expression system. The overexpressed proteins were subjected to various protein chromatographic methods, including the conventional Ni-NTA affinity chromatography and the size exclusion chromatography via FPLC system. The purified recombinant proteins were characterized for their identity and purity using SDS-PAGE. Two detection methods were used for SDS-PAGE, which were Coomassie Brilliant Blue staining and silver staining. UV-VIS spectrophotometry and Bradford assay were utilized to estimate the yield of purified protein from the expression culture, and the estimated yield was 2 mg of purified RecR protein per 1 liter of expression culture. SDS-PAGE analysis showed successful purification of the target proteins with a >95% purity. Future works will include improving the purity of the protein using different protein chromatographic methods and utilizing the purified proteins to further understand the oligomerization and the DNA binding mechanism of E. coli RecR. </p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="26"></a><h3>PGE1 VS PGE2 IN THE PROMOTION OF MSC DIFFERENTIATION TO OSTEOBLASTS</h3><p><b>Author(s):  </b><span>Sophie Lazar (Senior, Biology), Alisha  Malla (Senior, Biology), Dean  Koenig (Senior, Biology)</span></p><p><b>Advisor(s):  </b><span>Thomas Owen</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>Bone marrow mesenchymal stem cells (MSCs) are multipotent cells that have the capacity to differentiate into an assortment of cell types including osteoblasts, chondrocytes, myocytes, and adipocytes depending on external signals in their microenvironment.  These signals can include prostaglandins, which are small lipid compounds derived from arachadonic acid in the cell membrane. Two E-series prostaglandins, PGE1 and PGE2, increase MSC differentiation into osteoblasts through the activation of two G-protein coupled receptors, EP2 and EP4. The question remains as to why there are two structurally similar ligands that activate identical receptors and promote osteoblast differentiation.  In our work, the effects of various concentrations and exposure times of the two ligands were studied. Primary rat bone marrow cells were initially plated in MEMa with 15% FBS.  At the first media change, the cells were fed with MEMa with 15% FBS supplemented with 10 nM dexamethasone in order to promotes osteogenesis. Following 7-8 days of growth, some of the cells were assayed for alkaline phosphatase (AP) activity.  AP is commonly used as a marker for early differentiation as activity of this enzyme is known to increase with osteoblastic differentiation. The quantity and average size of mineralized cell nodules was then used as a later marker of osteoblast differentiation. Nodule analysis occurred 18-21 days after plating.  For nodule formation, cells were fed beginning on day 7 with differentiation medium (MEMa with 10% FBS, 10 nM dexamethasone, 50 ug/ml ascorbic acid, and 2 mM NaPhosphate). Although data are still being gathered, an important difference has already been noted; while PGE1 is more efficacious  (8-12 fold increase in AP activity), PGE2 is more potent, requiring a lower concentration to promote osteogenesis.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="27"></a><h3>QUANTITATIVE MEASURE OF VASCULAR SMOOTH MUSCLE RELAXATION IN PRIMARY RAT DESCENDING AORTA CELLS</h3><p><b>Author(s):  </b><span>Candace  Morales-Wilde (Junior, Biology/Chemistry), Brianna  Brianna Dziedzic (Senior , Biology )</span></p><p><b>Advisor(s):  </b><span>Thomas Owen</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>Prostaglandins are small lipids derived from arachadonic acid in the cell membrane through a series of enzymatic steps.  Prostaglandin E1 and E2 stimulate the differentiation of mesenchymal stem cells into osteoblasts and thus could be potential treatments for osteoporosis. Unfortunately, the interaction of either of these molecules with the cell surface receptor EP4 also results in a decrease in blood pressure through vasodilation.  It is possible that medicinal chemistry efforts could solve this problem, but an in vitro method for measuring vasodilation would be needed.  Our work is focused on preparing primary cultures of rat descending aortic smooth muscle cells via a published procedure, performing molecular characterization to ensure that the cells are genuine vascular smooth muscle cells, and then to develop and implement a method to stain the nuclei of the cells in order to measure any changes in their relative positions following PGE1 or PGE2 treatment. Characterization of the cells will be initially done by RT-qPCR for the α1B and α1D adrenergic receptor subtypes which have been shown to be expressed by vascular smooth muscle cells.  RNA from rat renal arteries will be used as a positive control. An antibody to α-SM-actin will be utilized to perform immunofluorescent staining of VSMCs in cell culture. VSMC nuclei will be stained using Hoechst dye, which makes it possible to view distances between individual VSMC nuclei as well as changes in the distance when VSMCs are exposed to PGE1 or PGE2.  An increase in nuclei distance would correspond to vasodilation and therefore a decrease in blood pressure. Sodium nitroprusside will be used as a positive control in experiments testing PGE2 and PGE1. It is a vasodilator that releases NO which activating guanylyl cyclase and forming cGMP, a molecule that inhibits calcium entry into the cell thereby inducing vascular smooth muscle relaxation.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="28"></a><h3>SILENCING THE KINASE PROTEIN C-YES</h3><p><b>Author(s):  </b><span>Chloe Veras (Junior, Biology)</span></p><p><b>Advisor(s):  </b><span>Thomas Owen</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>c-src is a non-receptor tyrosine kinase and a member of a gene family that is responsible for many biological functions.  The knock-out of the c-src gene in mice shows an osteopetrotic (too much bone) phenotype and further investigations over the last two decades have shown that this results from a combination of underactive osteoclasts and overactive osteoblasts.  The molecular mechanisms underlying the increase osteoblast activity are unclear, but could serve as a basis for treating people with low bone mass.  Other students have worked on this question using a compound (PP2) which decreases c-src activity but which also decreases the activity of the other c-src family kinases.  The purpose of our labs overall research project is to develop reagents through which c-src kinase activity can be specifically decreased in the absence of effects on other members of its gene family.  This will help us to begin to understand the signaling pathways through which it regulates osteoblast activity.  One important step in this work is to also develop negative control reagents, in this case, reagents which can specifically silence c-yes, the kinase most closely related to c-src. The purpose of this is to experimentally prove that even if the protein most closely related to c-src is shut down, there is no effect on the osteoblasts.  We have developed an RT-qPCR method to quantitate c-yes mRNA expression using a primer set which amplifies an approximately 360 bp fragment.  We have also designed a short hairpin RNA (shRNA) as a silencing reagent and are in the process of cloning it into a lentiviral transfer plasmid. Once this work silencing c-yes is completed, we will be able to confirm that c-src kinase affects bone directly and that even closely related members of its family do not.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="29"></a><h3>SOLVENT-FREE SYNTHESIS OF CHALCONES AS ANTIOXIDANTS</h3><p><b>Author(s):  </b><span>Joseph Szorentini (Senior, Biochemistry), Matthew Von Bargen (Senior, Biochemistry), Daniel Ferrante (Junior, Chemistry)</span></p><p><b>Advisor(s):  </b><span>Jay Carreon</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>Research has shown chalcones (1,3-diphenylprop-2-propen-1-ones) to display a wealth of biological properties-among them a very strong antioxidant activity. We want to understand how the chemical structure influences the antioxidant ability of these compounds. A small library of chalcones from various mixtures of aromatic benzaldehydes substituted with electron-donating and electron-withdrawing substituents at the para- position and acetophenone have been synthesized via the <i>Claisen-Schmidt</i> condensation in order to contribute to an extensive structure-activity relationship study. Although most aldol condensation reactions are run in organic solvents at elevated temperatures, solventless aldol condensations have recently been shown to be efficient. Solvents are a significant component of industrial chemical waste. Making a reaction solventless significantly reduces chemical waste and hazard. The synthesis and characterization will be discussed.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="30"></a><h3>SYNTHESIS OF HYDRAZONES AS ANTIBIOTICS </h3><p><b>Author(s):  </b><span>Ryan Ludvigsen (Senior, Biochemistry), Chris  Warren (Senior, Psychology), Sydney  Kauffman (Sophomore, Biology), Karina Borda (Sophomore, Biology), Jennifer Trabucco (Sophomore, Biochemistry)</span></p><p><b>Advisor(s):  </b><span>Jay Carreon</span></p><p class="type"><span>Presented in Session 2 and as a Poster</span></p><p class="abstract_text"></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="31"></a><h3>TESTING THE EFFICACY OF RNAI FEEDING IN C. ELEGANS</h3><p><b>Author(s):  </b><span>Matthew Von Bargen (Senior, Biochemistry), Samantha Davis (Junior, Biology)</span></p><p><b>Advisor(s):  </b><span>Joost Monen</span></p><p class="type"><span>Presented in Session 2 and as a Poster</span></p><p class="abstract_text"><p>RNA-mediated interference (RNAi) is a process by which double stranded RNA (dsRNA) molecules inhibit gene expression via specific degradation of mRNA transcripts. Since its discovery nearly 20 years ago, researchers have utilized this process to specifically knock-down genes of interest. Several techniques have been developed to induce RNAi in the nematode C. elegans. One technique involves injecting dsRNA directly into the C. elegans. Another technique requires feeding C. elegans with bacteria expressing dsRNA. This study, which focuses particularly on the feeding technique, assesses the effectiveness in knocking-down HCP-3, a CENP-A homolog critical to centromeric identity and thus chromosome segregation.  Knock-down effectiveness using the feeding technique will be measured via western blot and immunofluorescent staining and phenotypic consequences assessed via live imaging and an embryonic lethality assay.  Currently, the RNAi feeding and western blotting protocols have been implemented and the efficacy of the RNAi feeding has  been confirmed in all measurable assays.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="32"></a><h3>TESTING THE EFFICACY OF RNAI INJECTION IN C. ELEGANS</h3><p><b>Author(s):  </b><span>JJ Fritsch (Junior, Biology), Austin Marcelo (Sophomore, Biology)</span></p><p><b>Advisor(s):  </b><span>Joost Monen</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>RNA-mediated interference (RNAi) is a process by which RNA molecules inhibit gene expression via specific degradation of mRNA transcripts.  Since its discovery nearly 20 years ago, researchers have utilized this understanding to specifically knock-down genes of interest.  In the nematode C. elegans, several RNAi techniques have been developed, including injection of double stranded RNA (dsRNA) and feeding of bacteria expressing dsRNA.  In this study, we will test the injection method to knock-down an essential mitotic protein HCP-3, and assess the effectiveness of this strategy for use in future studies.  Knock-down effectiveness will be measured by western blot analysis and immunofluorescence, and the phenotypic consequences will be assayed by live-imaging of cell division in mCherry:Histone-H2B & GFP:α-tubulin transgenic worms.  Currently, we have synthesized the dsRNA and have begun injecting, which will, in the future, be used to rigorously test the efficacy of both injection and feeding.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="33"></a><h3>TRANSCRIPTOME WIDE ANALYSIS OF CIS-REGULATORY G-QUADRUPLEX MOTIFS AND CSTF BINDING SITES INVOLVED IN MAMMALIAN POLYADENYLATION</h3><p><b>Author(s):  </b><span>Haley Wight (Senior, Bioinformatics)</span></p><p><b>Advisor(s):  </b><span>Paramjeet Bagga, Scott Frees</span></p><p class="type"><span>Presented in Session 1 and as a Poster</span></p><p class="abstract_text"><p>Polyadenylation is a widespread post-transcriptional regulatory mechanism that can modulate the behavior of mRNA transcripts.  It is a tightly regulated process that plays important physiological roles in biological processes such as differentiation, proliferation, and tumorigenesis.  Polyadenylation-cleavage is achieved through formation of a multi-component protein complex that binds to specific sequences in the 3' UTR. Protein binding is affected by cis-regulatory elements. </p><p>The CStF protein of the Polyadenylation-cleavage complex binds to U-Rich Sequence (URS) elements for efficient 3' processing.  URS motifs consist of five bases; and their role in polyadenylation is reliant on the number of uracil residues and their distance from the polyA site. Previous studies have hypothesized that G-quadruplex structures and URS motifs work synergistically in regulated polyadenylation.  A G-quadruplexis a three-dimensional structure formed by guanine rich nucleic acids that can serve as a binding site for the hnRNPH' protein; a protein known to stimulate mRNA cleavage.  The goal of our study was toinvestigate synergism between G-quadruplexes and URS motifs in polyadenylation.</p><p>We studied the distribution of these motifs downstream of 26,585 distinct polyA sites and found that 28% of the polyA sites were associated with downstream G-quadruplexes.  We observed G-quadruplexes downstream of only 18.8% of U-rich motifs with five uracil residues.  As the motifs decreased in uracil residues, the percent of URS with a G-quadruplex downstream increased by a factor of two.  We considered a strong URS element as having four or five uracil residues in the region optimal for polyadenylation. Among the strong URS elements, only 24.4% had a downstream G-quadruplex. Conversely, 37.8% of weak URS sites had a downstream G-quadruplex. The prevalence of G-quadruplexes among weak URS motifs suggests that G-quadruplexes play an auxiliary role to aid weak URS elements. </p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="34"></a><h3>TRANSCRIPTOME WIDE DISTRIBUTION PATTERNS OF CONSERVED CIS-REGULATORY U-RICH SEQUENCE MOTIFS IN THE HUMAN POLYA REGIONS</h3><p><b>Author(s):  </b><span>Mingzhao Liu (Senior, Bioinformatics &amp; Computer Science)</span></p><p><b>Advisor(s):  </b><span>Paramjeet Bagga, Scott Frees</span></p><p class="type"><span>Presented in Session 1 and as a Poster</span></p><p class="abstract_text"><p>Polyadenylation is an essential process in the formation of the 3'-end of eukaryotic mRNAs. Polyadenylation determines the site of cleavage in the 3' untranslated region (UTR), protects the mRNA from degradation, and regulates gene expression. Cleavage is achieved through formation of a multi-component protein complex that binds to specific cis-regulatory sequences in the 3' UTR.</p><p>U-rich sequences (URS) are located downstream of the polyadenylation (polyA)/cleavage site and are known to bind to the Cleavage Stimulation Factor (CstF) protein of the cleavage complexfor efficient cleavage of the polyA site. The strength (number of uracils within a five nucleotide stretch) of the URS affects the efficiency of cleavage. As such, U-rich sequence elements are considered important regulatory elements in polyadenylation. Alterations of URS have been demonstrated to cause adverse effects on the process of polyadenylation and change the cleavage site's positioning. Previous 'wet-bench' research performed on two mRNAs (E2A, SV40) found that optimal URS were located either 19-27 or 15-30 nt downstream of the polyA site.</p><p>The purpose of our study was to determine if the findings from the E2A and SV40 can be extended to the entire human transcriptome. </p><p>By comparing the distance of URS from polyA sites in homologous mRNAs, we have determined that URS tend to occur within a region 22-33 nucleotides downstream of the polyA site.  This region contains a quantity of URS more than two standard deviations above the mean. Of the 27,005 human mRNAs with conserved URS, 90.6% contain a URS within the region, accounting for 90.3% of genes with a conserved URS.  Using A, C, and G rich sequences as negative controls, we demonstrate that this distribution is unique to URS, further supporting the hypothesis that this location is optimal for URS function in the human transcriptome. </p> </p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="35"></a><h3>TREE MORTALITY, CANOPY OPENINGS AND PLANT SPECIES COMPOSITION IN A HARDWOOD FOREST OF THE NEW JERSEY HIGHLANDS</h3><p><b>Author(s):  </b><span>Michael Finocchio (Senior, Environmental Science), Stuart Brown (Senior, Environmental Science), Ashley Olsen (Senior, Environmental Science)</span></p><p><b>Advisor(s):  </b><span>Eric Wiener</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>Tree mortality rates have accelerated in the New Jersey Highlands due to severe weather and biological agents (e.g., gypsy moths, hemlock woolly adelgids, beech bark disease). The purpose of this study was to examine tree mortality and how the subsequent canopy openings are impacting forest structure and plant species composition in a 16.3 hectare forest plot. Canopy gaps &ge; 7m in length were measured, and the species, trunk diameter and likely cause of mortality were recorded for the dead trees that created the gaps. Vegetative cover was characterized for each of five vertical strata, and living trees were identified and measured within 7.5 m radius plots centered in the middle of the gaps, as well as in nearby areas where no large trees had died within approximately the past 5 years.  </p><p>Forty-nine canopy openings &ge; 7m in length and caused by mortality of a total of 78 canopy and subcanopy trees occupied 4% of the study plot. Relative densities of living red oak and black oak trees were significantly lower in canopy gaps than predicted by the relative densities of dead conspecifics, which suggests that populations of these species are in decline. Relative density of sugar maple was significantly greater in gaps than in undisturbed plots, suggesting that sugar maple recruitment is accelerating. Conversely, relative densities of red maple and American beech were significantly greater in undisturbed plots than in gaps. In addition, percent ground covered by invasive species was significantly greater in large gaps than in small gaps or undisturbed plots, and was inversely correlated with cover estimates for the lower and upper midstory strata. Overall, results from this study suggest that accelerated rates of tree mortality are in part causing shifts in plant species composition in the study forest, with a particularly notable decline in red oak and likely increases in sugar maple and nonnative understory plants.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div><div class="abstract"><a name="36"></a><h3>TWO FALL SEASONS OF RAPTOR MIGRATION DATA FROM THE RAMAPO MOUNTAINS: HELPING UNDERSTAND REGIONAL TRENDS</h3><p><b>Author(s):  </b><span>Zachary Batren (Senior, Environmental Science), Patricia Goense (Senior, Environmental Science), Dylan Keastead (Senior, Environmental Science), Lawrence Kelsey (Senior, Environmental Science), Lichtenberger Courtney (Senior, Environmental Science)</span></p><p><b>Advisor(s):  </b><span>Eric Wiener</span></p><p class="type"><span>Poster Only</span></p><p class="abstract_text"><p>While much of our understanding of raptor migration along North America's Atlantic flyway comes from storied, long-running hawk watches on the Kittatinny Ridge and along the Atlantic coast, a plethora of additional hawk watches that have been established in recent decades has allowed for more extensive coverage of the region. The purpose of this ongoing study is to examine whether collecting data on migrating raptors from the Ramapo Mountains (eastern region of the New Jersey Highlands) can complement efforts at three other nearby hawk-watch sites to all together provide fairly complete coverage of raptor migration through a relatively narrow geographic region that lies between the Kittatiny Ridge and the Atlantic coast.  </p><p>A combined total of 19,422 migrants of 15 raptor species were observed from the Ramapo Mountains during the Fall migration seasons of 2014 and 2015. Despite logging only 406 observation hours, which is under half of the hours logged at each of the other three sites, the total raptor count from the Ramapo Mountains was the second highest among the sites. The Ramapo Mountains, therefore, are clearly an excellent location for studying raptor migration through the region. Daily counts of many species often differed dramatically among the sites, suggesting that data from all four sites are essential for monitoring raptor migration through the area. Surprisingly, initial analyses suggest that weather variables that often are good predictors of migrant raptor counts along the Kittatiny Ridge and at coastal hawk-watching sites did not tend to be useful for understanding daily variation in migrant raptor counts in the region reported here. Consequently, further research and analysis to attempt to unravel the complex nature by which raptors choose specific migration pathways seems warranted.</p></p><p class="back"><a href="#contents">Back to contents</a></p></div></body></html>